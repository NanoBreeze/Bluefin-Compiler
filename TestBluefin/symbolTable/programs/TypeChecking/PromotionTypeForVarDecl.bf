int f() {}
float g() {}

int main() {
	int a = f();
	float b = f();
	float c = g();
	float d = 8;
	bool z;
}