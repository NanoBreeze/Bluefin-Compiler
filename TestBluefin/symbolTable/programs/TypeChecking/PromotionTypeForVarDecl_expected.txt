f - t_int - t_int
f() - t_int - t_int
f - t_int - t_int
f() - t_int - t_float
g - t_float - t_float
g() - t_float - t_float
8 - t_int - t_float
